### PR TITLE
Add an MNIST Example with Keras Tuner

### PR DESCRIPTION
**Pull request by ostamand**
^

### CODE REVIEW

File: benchmarks/README.md [hunk 1]
```diff
@@ -0,0 +1,56 @@
...
+```
+
+where `--model_dir` is where the model was saved after training.
+
+Results obtained on the test loss are summarized in the table below (trained only 21k steps VS 130k for the reference):
+
+| Model | Test Accuracy |
+| -- | -- |
+| Reference | 96.99 |
+| `TabNetClassifier` |  96.59 |
```
Comment (NicoCoallier) [inline benchmarks/README.md]: Is this post hyper parameters tuning ?

Comment (ostamand) [reply benchmarks/README.md]: Oups this file is actually for the convertype dataset (ie my PR is messy lol) & I did not do hyperparam tuning for this one. Used the same hyperparam used in the original paper to compare results. The results above are for only 20k steps VS 130k steps for the paper. I will update with the full training later.
